homeassistant/components/plex/__init__.py

```diff
@@ -140,10 +141,13 @@ def _setup_plex(hass, config):
     cancel = hass.data[PLEX_DOMAIN][REFRESH_LISTENERS].pop(server_id)
     await hass.async_add_executor_job(cancel)
 
-    for platform in PLATFORMS:
-        hass.async_create_task(
+    tasks = [
+        asyncio.ensure_future(
```

Comment:
This isn't necessary. Just collect the coroutine instances in a list and pass the tasks to asyncio.gather.

A coroutine function when called will return a coroutine object. When that object is awaited the coroutine will be run.

https://docs.python.org/3/library/asyncio-task.html#asyncio.gather